thing-url-adapter.js

```diff
@@ -38,17 +50,17 @@ class ThingURLProperty extends Property {
       method: 'PUT',
       headers: {
         'Content-type': 'application/json',
+        'Accept': 'application/json',
       },
       body: JSON.stringify({
         [this.name]: value
       })
-    }).then(res => {
-      return res.json();
-    }).then(response => {
```

Comment:
I put this logic here to handle the case where the web thing denies the property change (although I guess it could deny the change by throwing an error). It's also possible that the web thing limits the property change, i.e. 0 initial, 150 requested, 100 actual if it has an undocumented (or temporary) 100 max

Reply:
I know, but according to the spec, the PUT doesn't return anything, so this was breaking.

https://iot.mozilla.org/wot/#property-resource

Reply:
Oh, that's strange. The Gateway implementation does include the updated value.

@benfrancis What's your opinion here?

Reply:
Maybe the response to the PUT request should echo back the value. If that's what the gateway already does we can change the spec. The spec should maybe also define error conditions like permission denied and invalid value for example.